shell-control/agent/backend/shellcontrol.py

```
@@ -0,0 +1,181 @@
+        rprint(f"Request from the user: [bold]\"{user_prompt}\"[/bold]")
+        rprint("[bold red]Press Ctrl+C to stop iteration at any step.[/bold red]")
+
+        async for event in agent.process_user_request(user_prompt):
+            if event["type"]:
+                match event["type"]:
+                    case Event.AI_RESPONSE:
+                        rprint("\n[bold]=== AI Response ===[/bold]")
+                        pprint(event["payload"], expand_all=True)
+                    case Event.TOOL_SUCCESS:
+                        rprint("\n[bold]=== Tool Output ===[/bold]")
+                        if event["payload"]["returncode"] != None:
```

Ruff (0.8.2)

Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)